h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLBuildSpecV99.java

```diff
@@ -255,6 +254,10 @@ V getFormattedValue() {
     @API(help="The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).",
             level = API.Level.expert)
     public StepDefinitionV99[] modeling_plan;
+    
+    @API(help="The list of preprocessing steps to run. Only 'targetencoding' is currently supported.",
```

Comment:
```suggestion
    @API(help="The list of preprocessing steps to run. Only 'target_encoding' is currently supported.",
```

Reply:
@sebhrusen Did you want to change the Java parameter definition to use `"target_encoding"` instead of `"targetencoding"` too?  Or is the value hardcoded to "targetencoding" on the back-end?